src/Rule/IsTrue.php

```diff
@@ -28,26 +33,57 @@ final class IsTrue implements RuleWithOptionsInterface, SkipOnErrorInterface, Wh
     use SkipOnErrorTrait;
     use WhenTrait;
 
+    /**
+     * @const Default message used for all cases.
+     */
+    private const DEFAULT_MESSAGE = 'The value must be "{true}".';
+
+    /**
+     * @param scalar $trueValue The value that is considered to be "true". Only scalar values (either int, float, string
+     * or bool) are allowed. Defaults to `1` string.
+     * @param bool $strict Whether the comparison to {@see $trueValue} is strict:
+     *
+     * - Strict mode uses `===` operator meaning the type and the value must both match to those set in
+     * {@see $trueValue}.
+     * = Non-strict mode uses `==` operator meaning that type juggling is performed first before the comparison. You can
+     * read more in the PHP docs:
+     *
+     * - https://www.php.net/manual/en/language.operators.comparison.php
+     * - https://www.php.net/manual/en/types.comparisons.php
+     * - https://www.php.net/manual/en/language.types.type-juggling.php
```

Comment:
Use @link annotation?

Reply:
Didn't use them here because they will be inline. But OK, let's add them as inline.

Reply:
Didn't use it here because of the list and the limitation to use inline tag only. But OK, let's add them as inline.